pcsx2/Counters.cpp

```diff
@@ -515,6 +520,7 @@ static __fi void VSyncEnd(u32 sCycle)
 		sioNextFrame();
 
 	frameLimit(); // limit FPS
+	gsPostVsyncStart(); // MUST be after framelimit; doing so before causes funk with frame times!
```

Comment:
Move these 2 lines to the beginning of static __fi void VSyncStart(u32 sCycle)
Needs to be on the start function to fix some graphical errors